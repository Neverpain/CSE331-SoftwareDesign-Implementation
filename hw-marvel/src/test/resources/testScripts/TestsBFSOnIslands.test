# Tests the makeGraph method on the islands file to see that it is
# built correctly, prints out paths of some nodes

LoadGraph g islands.csv

FindPath g Victoria Java

FindPath g Java Victoria

FindPath g Honshu Borneo

FindPath g Baffin Honshu

FindPath g Java Greenland

FindPath g Java Java