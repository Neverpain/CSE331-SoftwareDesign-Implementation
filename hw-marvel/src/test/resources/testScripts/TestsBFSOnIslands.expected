# Tests the makeGraph method on the islands file to see that it is
# built correctly, prints out paths of some nodes

loaded graph g

path from Victoria to Java:
no path found

path from Java to Victoria:
no path found

path from Honshu to Borneo:
no path found

path from Baffin to Honshu:
no path found

path from Java to Greenland:
no path found

path from Java to Java:

